src/test/java/teammates/test/cases/common/CommentAttributesTest.java

```diff
@@ -96,11 +96,12 @@ public void testValidate() {
                 commentText
                 );
         
+        // There are 4 recipients in total
         List<String> expectedErrorMessage = new ArrayList<String>();
-        expectedErrorMessage.add(String.format(FieldValidator.EMAIL_ERROR_MESSAGE, incorrectEmail, FieldValidator.REASON_INCORRECT_FORMAT));
-        expectedErrorMessage.add(String.format(FieldValidator.EMAIL_ERROR_MESSAGE, "recipient-1", FieldValidator.REASON_INCORRECT_FORMAT));
-        expectedErrorMessage.add(String.format(FieldValidator.EMAIL_ERROR_MESSAGE, "recipient-3", FieldValidator.REASON_INCORRECT_FORMAT));
-        expectedErrorMessage.add(String.format(FieldValidator.EMAIL_ERROR_MESSAGE, "recipient-2", FieldValidator.REASON_INCORRECT_FORMAT));
+        expectedErrorMessage.add(FieldValidator.EMAIL_ERROR_MESSAGE);
+        expectedErrorMessage.add(FieldValidator.EMAIL_ERROR_MESSAGE);
+        expectedErrorMessage.add(FieldValidator.EMAIL_ERROR_MESSAGE);
+        expectedErrorMessage.add(FieldValidator.EMAIL_ERROR_MESSAGE);
```

Comment:
Because invalid inputs no longer displayed, this becomes 4 identical message. If the other portions are okay, I'll proceed to refactor this test.